core/src/main/java/tech/beshu/ror/requestcontext/SerializationTool.java

```diff
@@ -45,70 +50,94 @@
     zuluFormat.setTimeZone(TimeZone.getTimeZone("GMT"));
   }
 
-  public SerializationTool() {
+  private final LoggerShim logger;
+  private Optional<AuditLogSerializer> customSerializer;
```

Comment:
IMHO this should be: 
`private final AuditLogSerializer auditLogSerializer;`
and should either be set to the default serializer (that returns  AuditLogContext) or to the custom serializer (if specified)

Reply:
Yes that would be more clean 👍